server/controllers/user.js

```diff
@@ -80,3 +81,15 @@ export const logout = (req, res) => {
 
   return res.json({});
 };
+
+export const current = (req, res) => {
+  const { [TOKEN_COOKIE_NAME]: token } = req.signedCookies;
+
+  if (!token) {
+    return res.status(401).json({ error: STATUS_CODES[401] });
+  }
+
+  const { user } = verifyToken(token);
```

Comment:
`verifyToken(token)` will throw an error if token is expired or malformed, we need to handle